src/Microsoft.Identity.Web/AuthorityHelpers.cs

```diff
@@ -33,5 +33,22 @@ internal static string EnsureAuthorityIsV2(string authority)
 
             return authority;
         }
+
+        internal static string? BuildCiamAuthorityIfNeeded(string? authority)
```

Comment:
maybe make a comment linking to a GitHub issue that this will be removed after CIAM GA?

Reply:
done